### PR TITLE
runfix: Use opacity instead of TransitionContainer on the LoadingSpinner

### PR DESCRIPTION
**Pull request by ffflorian**
Since we removed only the `FlexBox` and the Loading Spinner itself after the page is loaded, we had a left-over element over the whole interface which rendered the app useless :slightly_smiling_face:

### CODE REVIEW

File: electron/renderer/src/components/LoadingSpinner.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+#loading-spinner-wrapper {
+  display: flex;
+  height: 100vh;
+  position: absolute;
+  width: 100%;
+  z-index: 99;
+  transition: opacity 0.25s ease-in-out;
+  -moz-transition: opacity 0.25s ease-in-out;
+  -webkit-transition: opacity 0.25s ease-in-out;
```
Comment (thisisamir98) [inline electron/renderer/src/components/LoadingSpinner.css:28]: does not compiler add these automatically?

File: electron/renderer/src/components/LoadingSpinner.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+#loading-spinner-wrapper {
```
Comment (thisisamir98) [inline electron/renderer/src/components/LoadingSpinner.css:20]: if you're going to use this multiple times in a page I suggested to not use an ID and use classes instead but if it's just one place it's fine...

**Review by thisisamir98 — APPROVED** [review]

File: electron/renderer/src/components/LoadingSpinner.css (hunk 1 above)
Comment (ffflorian) [reply electron/renderer/src/components/LoadingSpinner.css:28]: What compiler? I am using css :)

File: electron/renderer/src/components/LoadingSpinner.css (hunk 2 above)
Comment (ffflorian) [reply electron/renderer/src/components/LoadingSpinner.css:20]: Mh you are right, we may be using it in multiple webviews..

File: electron/renderer/src/components/LoadingSpinner.css (hunk 1 above)
Comment (Yserz) [reply electron/renderer/src/components/LoadingSpinner.css:28]: Bardia means something like postCSS as a CSS pre-compiler.

Comment (ffflorian) [reply electron/renderer/src/components/LoadingSpinner.css:28]: Ah okay, we don't have that in wire-desktop.

File: electron/renderer/src/components/LoadingSpinner.css [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+ */
+
+.loading-spinner-wrapper {
+  display: flex;
+  height: 100vh;
+  position: absolute;
+  width: 100%;
+  z-index: 99;
+  transition: opacity 0.25s ease-in-out;
+  -moz-transition: opacity 0.25s ease-in-out;
```
Comment (bennycode) [inline electron/renderer/src/components/LoadingSpinner.css:27]: Wire desktop runs a Chromium, so we don't need the `-moz-` prefix, or am I wrong?

Comment (ffflorian) [reply electron/renderer/src/components/LoadingSpinner.css:27]: True!

**Pull request merged**